src/classes/HH_Container_LCTRL.cls

```diff
@@ -347,16 +347,13 @@ public with sharing class HH_Container_LCTRL {
             TDTM_ProcessControl.setRecursionFlag(TDTM_ProcessControl.flag.HH, true);
             
             // allow potential duplicates when rules to warn are turned on.
-            Database.DMLOptions dml = new Database.DMLOptions();
-            dml.DuplicateRuleHeader.AllowSave = true;
-
-            list<Database.SaveResult> listSR = Database.insert(listInsert, dml);
+            List<Database.SaveResult> listSR = UTIL_DMLService.insertRecords(listInsert);
```

Comment:
@mpusto @joy-sfdo another example scenario where we were explicitly allowing the save regardless of duplicate warning, in ManageHouseholdUI.  do we want this behavior changed to use the setting, which is defaulting to false?

Reply:
@davidhabib -- Same as Contact Merge, will set AllowSave to true.